src/components/StakingPage/VotingPower/index.tsx

```diff
@@ -3,6 +3,7 @@ import { jsx } from 'theme-ui';
 import styles from './style';
 import images from '../../../images/index';
 import ReactTooltip from 'react-tooltip';
+import { STAKING_VERSION } from '../../../containers/VotingPowerContainer';
```

Comment:
naming error
STAKING_VERSION is like a constants
should use 'StakingVersion' for enum